### PR TITLE
DOCS-283 - Update tour new UI article for Kanso Beta 2.0

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request updates the following article to conform to the new UI for the Kanso Beta 2.0 release: 
https://help.sumologic.com/docs/get-started/sumo-logic-ui-new/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-283](https://sumologic.atlassian.net/browse/DOCS-283)

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, could you review this PR, since you are our Kanso lead and you wrote the original article?

File: docs/get-started/sumo-logic-ui-new.md [hunk 1]
```diff
@@ -82,40 +82,79 @@ The left nav panel provides easy access to libraries, searches, folders, and y
...
-
-If you're an admin, you can view the Library in [Content Administrator](#content-administrator-library) mode.
+   * Dashboards and searches shared within your organization.<br/><img src={useBaseUrl('/img/get-started/library.png')} alt="library.png" style={{border: '1px solid gray'}} width="400"/> <br/>Click **Open library page** to use the **Library** search bar to find items quickly. **View as** mode should be set to **Me**.<br/><img src={useBaseUrl('img/get-started/library-search.png')} alt="library-search.png" style={{border: '1px solid gray'}} width="700"/> <br/>If you're an admin, you can view the Library in [Content Administrator](#content-administrator-library) mode.
+* **Recent** dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/recent.png')} alt="recent.png" width="400"/>
+* Your **Favorites** list of favorited dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/favs.png')} alt="favs.png" style={{border: '1px solid gray'}} width="400"/>
```
Comment (kimsauce) [inline docs/get-started/sumo-logic-ui-new.md:87]:
```suggestion
* Your **Favorites** list of favorited dashboards and searches.<br/><img src={useBaseUrl('/img/get-started/favs.png')} alt="favs.png" style={{border: '1px solid gray'}} width="400"/>

Use the **Library** search bar to find the above quickly. **View as** mode should be set to **Me**.<br/><img src={useBaseUrl('img/get-started/library-search.png')} alt="library-search.png" style={{border: '1px solid gray'}} width="550"/>

If you're an admin, you can view the Library in [Content Administrator](#content-administrator-library) mode.
```

File: docs/get-started/sumo-logic-ui-new.md [hunk 2]
```diff
@@ -82,40 +82,79 @@ The left nav panel provides easy access to libraries, searches, folders, and y
...
-Enlarge your working area by hiding the left-side nav panel. Just click the hamburger menu icon. To unhide it, click the hamburger menu icon again.<br/><img src={useBaseUrl('/img/get-started/hamburger.gif')} alt="hamburger.gif" width="350"/>
+Enlarge your working area by hiding the left-side nav panel. Just click the hamburger menu icon. To unhide it, click the hamburger menu icon again.<br/><img src={useBaseUrl('/img/get-started/hamburger.gif')} alt="hamburger.gif" width="300"/>
 :::
 
 ## Using the top navigation bar
 
-### Access Apps, Admin settings
+### Access settings
+
+The global toolbar (top nav bar) provides you access to (from left to right): [Go To...](#go-to-all-menu-options), [Help](/docs/get-started/help), [Configuration](/docs/send-data), [Administration](/docs/manage), and your user profile options ([Notifications](/docs/get-started/account-settings-preferences) and [Preferences](/docs/get-started/account-settings-preferences)).
```
Comment (kimsauce) [inline docs/get-started/sumo-logic-ui-new.md:97]:
```suggestion
The global toolbar (top nav bar) provides access to various functions in the following order: [Go To...](#go-to-all-menu-options), [Help](/docs/get-started/help), [Configuration](/docs/send-data), [Administration](/docs/manage), and your user profile options ([Notifications](/docs/get-started/account-settings-preferences) and [Preferences](/docs/get-started/account-settings-preferences)).
```

File: docs/get-started/sumo-logic-ui-new.md [hunk 3]
```diff
@@ -82,40 +82,79 @@ The left nav panel provides easy access to libraries, searches, folders, and y
...
+
+<img src={useBaseUrl('/img/get-started/admin-config.png')} alt="admin-config.png" style={{border: '1px solid gray'}} width="350"/>
+
+#### Go to all menu options
```
Comment (kimsauce) [inline docs/get-started/sumo-logic-ui-new.md:101]:
```suggestion
#### Go To... menu options
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
A few suggestions. Thanks for doing this! I've also cropped the replacement images to show less UI elements. This reduces the need to go in and update them again for future UI changes.

File: docs/get-started/sumo-logic-ui-new.md (hunk 1 above)
Comment (jpipkin1) [reply docs/get-started/sumo-logic-ui-new.md:87]: @kimsauce, I already moved this content up under the Library bullet.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce. I took most of your suggestions, except for the one about the Library, since I'd already moved the content under the Library bullet.

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Pushing this live today per Kevin K.

**Pull request merged**